src/types/todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './user';
+
+export type Todo = {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number
+  user?: User | null
```

Comment:
```suggestion
  user: User | null
```
You don`t need '?' in this task